src/Fixer/Operator/BinaryOperatorSpacesFixer.php

```diff
@@ -29,66 +29,147 @@
  */
 final class BinaryOperatorSpacesFixer extends AbstractFixer implements ConfigurationDefinitionFixerInterface
 {
+    use AlignmentFixerHelperTrait;
+
+    /**
+     * @internal
+     */
+    const SINGLE_SPACE = 'single_space';
+
+    /**
+     * @internal
+     */
+    const ALIGN = 'align';
+
+    /**
+     * @internal
+     */
+    const ALIGN_SINGLE_SPACE = 'align_single_space';
+
+    /**
+     * @internal
+     */
+    const ALIGN_SINGLE_SPACE_MINIMAL = 'align_single_space_minimal';
+
+    /**
+     * @internal
+     * @const Placeholder used as anchor for right alignment.
+     */
+    const ALIGN_PLACEHOLDER = "\x2 ALIGNABLE%d \x3";
+
+    private static $allowedValues = [
+        self::ALIGN,
+        self::ALIGN_SINGLE_SPACE,
+        self::ALIGN_SINGLE_SPACE_MINIMAL,
+        self::SINGLE_SPACE,
+        null,
+    ];
+
+    /**
+     * @var array
+     */
+    private static $supportedOperators = [
+        '=',
+        '*',
+        '/',
+        '%',
+        '<',
+        '>',
+        '|',
+        '^',
+        '+',
+        '-',
+        '&',
+        '&=',
+        '&&',
+        '||',
+        '.=',
+        '/=',
+        '=>',
+        '==',
+        '>=',
+        '===',
+        '!=',
+        '<>',
+        '!==',
+        '<=',
+        'and',
+        'or',
+        'xor',
+        '-=',
+        '%=',
+        '*=',
+        '|=',
+        '+=',
+        '<<',
+        '<<=',
+        '>>',
+        '>>=',
+        '^=',
+        '**',
+        '**=',
+        '<=>',
+        '??',
+    ];
+
+    /**
+     * @var TokensAnalyzer
+     */
+    private $tokensAnalyzer;
+
     /**
-     * @var AbstractAlignFixerHelper[]
+     * @var array<string, string>
      */
-    private $alignFixerHelpers = [];
+    private $alignOperatorTokens = [];
```

Comment:
what is the value? string / token ?

Reply:
` @var array<string, string>`
`token type, alignment strategy`